website/content/partials/telemetry-metrics/vault/quota/lease_count/counter.mdx

```diff
@@ -0,0 +1,5 @@
+### vault.quota.lease_count.counter ((#vault-quota-lease_count-counter))
+
+Metric type | Value | Description
+----------- | ----- | -----------
+gauge       | lease | Total number of current leases
```

Comment:
These lease-count quota metrics appear to use the `name` of the quota rule as a label.  So this is really: number of leases counted by this quota rule.  In other words, if there 50 max leases defined by the quota rule, and there are currently 40 leases that are in scope of that quota rule, the value of this gauge will be 40.  There could still be 1000s of leases not in scope of this rule, or possibly not in scope of any rules.

Reply:
Clarification added